src/components/messageSpace/messageSpace.tsx

```diff
@@ -9,9 +9,13 @@ import Icon from '../icon'
 import MessageCanvas, {
   type MessageContainerProps,
 } from '../messageCanvas/messageCanvas'
-import type { ComponentMap, ThreadableMessage } from '../types'
+import type { ComponentMap, ThreadableMessage, WebSocketClient } from '../types'
 
 export interface MessageSpaceProps extends MessageContainerProps {
+  /** WebSocket connection to send and receive messages to and from a backend. Some components in this library require this value in order to be functional, such as the `Question` component. */
+  ws: WebSocketClient
+  /** ID of the current user. Some components in this library require this value in order to be functional, such as the `Question` component. */
+  sender: string
```

Comment:
```suggestion
  /** ID of the current user. This can be useful for component interactions, where current user id may be required */
  sender: string
```